client/components/namespace-navigation.vue

```diff
@@ -83,17 +83,23 @@ export default {
   components: {
     'detail-list': DetailList,
   },
-  created() {
+  async created() {
     this.namespaceDescCache = {};
 
-    this.getNamespaces().then(() => {
-      if (this.recentNamespaces.length == 1) {
-        const namespace = this.recentNamespaces[0];
-        const url = this.namespaceLink(namespace);
+    const { routingConfig } = await this.$http(`/api/web-settings`);
+    if (routingConfig?.defaultToNamespace) {
+      const url = this.namespaceLink(routingConfig.defaultToNamespace);
+      this.$router.push(url);
+      return;
+    }
```

Comment:
i think this is very indirect. i would prefer to set and store this inside the UI, instead of setting it in the server side config. what were the requirements for this? i dont have context

Reply:
This comes from saas requirements to be controlled by ops